src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -74,8 +76,16 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeFramework Include="GSS" />
     </ItemGroup>
 
+    <Message Text="IlcPath = $(IlcPath)" />
+    <Message Text="IlcHostPackagePath = $(IlcHostPackagePath)" />
+
+    <Exec Command="$(IlcPath)/source_code/libs/System.Globalization.Native/build-nativeaot-shim-standalone.sh $(IlcPath)/../ $(IntermediateOutputPath)" Condition="'$(LocalSystemGlobalizationNative)' != '' and '$(IlcPath)' != ''"/>
```

Comment:
We should use either `IlcHostPackagePath` everywhere or `IlcPath` everywhere. I do not see the reason why we would want to switch back and forth.

Reply:
@jkotas is `IlcHostPackagePath` still set even if you use `<PublishAot>true</PublishAot>` in your project file and don't reference the ILCompiler NuGet package?